dm/master/server.go

```diff
@@ -1134,7 +1136,7 @@ func checkAndAdjustSourceConfig(ctx context.Context, cfg *config.SourceConfig) e
 	if err = cfg.Adjust(ctx, fromDB.DB); err != nil {
 		return err
 	}
-	if _, err = cfg.Yaml(); err != nil {
+	if _, err := cfg.Yaml(); err != nil {
```

Comment:
Why change this here?

Reply:
I'm not sure what the best practice is here and under what circumstances `err` can be reused🤔